src/cephadm/cephadm

```diff
@@ -3305,8 +3328,12 @@ def command_bootstrap():
         ])
 
     if mon_network:
-        logger.info('Setting mon public_network...')
+        logger.info(f"Setting mon public_network to {mon_network}")
         cli(['config', 'set', 'mon', 'public_network', mon_network])
+    
+    if cluster_network:
+        logger.info(f"Setting mon cluster_network to {cluster_network}")
+        cli(['config', 'set', 'mon', 'cluster_network', cluster_network])
```

Comment:
`cluster_network` is only for OSD nodes so it doesn't make sense to set this at mon level.

Reply:
ack - should have been global!